### ISSUE TITLE
Android sign in is intermittently blocked by a white square

### ISSUE THREAD

**Issue by lncramer** [invertase/react-native-apple-authentication#283]
**Library Version: 2.2.0
Device: Pixel 4 XL
Android Version: 12**

Hey there,

I'm finding that ~90% of the time calling `await appleAuthAndroid.signIn()` as described in the docs correctly opens the webview, and I can see the Apple sign in page, but there is a little white square (super technical term 😉 ) in the center of the screen which prevents me from doing anything else.

When this happens I cannot even dismiss the square by going back or tapping outside of it on the screen - I am forced to completely close and re-open the app.

It's unclear to me why it rarely works - in these cases the white square still briefly appears, but it immediately goes away and allows me to interact with the Apple sign in page once the webview loads.

## Recordings
**Showing both the success and failure cases**

### Failure Case
**White square does not go away, prevents me from doing anything once the sign in screen loads**

https://user-images.githubusercontent.com/3388083/172723956-cb545d1f-8f49-4a2e-ab25-291ca7f3aeda.mp4

### Success Case
***White square goes away immediately, and I can interact with the sign in screen once it loads***

https://user-images.githubusercontent.com/3388083/172725408-08a3db61-e45a-4ae4-aa54-2732a4fb1a1a.mp4

**Comment by mikehardy** [comment]
That is really odd! Have you tried using Chrome WebView Debugging on a device that's failing and inspecting the webview?

https://developer.chrome.com/docs/devtools/remote-debugging/webviews/
https://stackoverflow.com/questions/21903934/how-to-debug-webview-remotely
https://github.com/react-native-webview/react-native-webview/blob/master/docs/Debugging.md#android--chrome

I've had great success with this in an unrelated pure-java android app, it might give you the clues needed to debug further

**Comment by lncramer** [comment]
Hey @mikehardy, thanks for your response! I tried out the Chrome WebView Debugging and it revealed that the white square did _not_ seem to be part of the webview code at all.

Searched around a bit more and found another post from someone who sounded like they had the same white square https://discussions.apple.com/thread/8516536. The solution there was just to restart the device, and after restarting my phone the issue seems to be fixed now!

So I think it's safe to say this is unrelated to this library. Closing this issue!
